src/sentry/api/endpoints/organization_recent_searches.py

```diff
@@ -59,10 +59,17 @@ def get(self, request, organization):
                 status=400,
             )
 
+        query_kwargs = {
+            'organization': organization,
+            'user': request.user,
+            'type': search_type,
+        }
+
+        if 'query' in request.GET:
+            query_kwargs['query__icontains'] = request.GET['query']
```

Comment:
Should we escape any literal `%` that users give us?

Reply:
The ORM handles that for us (assume you're just concerned about them adding wildcards to the middle?)

Reply:
Yeah. I would expect that I could search for a literal % without having to send %%.

Reply:
Yeah, I just confirmed that it works as expected. I don't really want to commit a test for it since this is just standard ORM behaviour.